core/src/test/java/nu/yona/server/templates/TemplateTest.java

```diff
@@ -61,19 +61,6 @@ public void testAppleAppSiteAssociation()
 	}
 
 	@Test
-	public void testAppleMobileConfig()
-	{
-		Context ctx = ThymeleafUtil.createContext();
-		String ldapUsername = "DummyLdapUserName";
-		String ldapPassword = "DummyLdapPassword";
-		ctx.setVariable("ldapUsername", ldapUsername);
-		ctx.setVariable("ldapPassword", ldapPassword);
-
-		String result = otherTemplateEngine.process("apple.mobileconfig.xml", ctx);
-		assertThat(result, containsString("<string>" + ldapUsername + "\\n" + ldapPassword + "</string>"));
```

Comment:
This test asserted that the `\n` was present between the LDAP username van LDAP password. Have you checked that this is still the case? Maybe move it to the integration test above?